### DISCUSSION TITLE
Can I copy maps from Maps.Me into OrganicMaps in Android?

### DISCUSSION THREAD

**Discussion by marc-hugo** [organicmaps/organicmaps#1542] [Ideas]
As per the title: can I copy Maps.Me country files from my Android sd card into the files folder of OrganicMaps? Would save me a lot of time. 

Reason why I ask this is, that I see that many country maps in the folder of Maps.Me have a different size than the ones I downloaded with OrganicMaps. Some are smaller, others are bigger. Maybe the maps contain different data?

Marc.

**Comment by biodranik** [comment]
Our maps are fresher, why do you need to use older maps?
You may try to move them, but we don't promise that it will work, because we already added some new object types which are absent in Maps.Me.

As we plan to further develop and update maps format, there are no plans to maintain the compatibility.

**Reply by marc-hugo** [reply]
Thanks biodranik, is clear. I'll download the maps via OM.
Where can I find the list of new object types?

OT question: where does OM store kmz pois? I have placed my Maps.Me pois in OM (via email), but do not see them in the OM files folder of my Android (not on sd card, not in the phone memory)

**Reply by biodranik** [reply]
Bookmarks are now not accessible on Android due to Android 11 restrictions. Why do you need to access them?

P.S. We plan to add "Export all bookmarks" button and then add a cloud sync/backup for bookmarks.

**Comment by darkguset** [comment]
Where does OM save the maps? (storage location) on the phone?

**Reply by biodranik** [reply]
You need root to access it: /data/data/app.organicmaps

Why do you need it?

**Reply by darkguset** [reply]
Thanks Alex,

But it doesn't seem to be there. I am using Magisk - perhaps it is copied somewhere else?
What is the extension name for the maps files?

From time to time I wipe the phone or try a new app and delete settings and maps. It is A LOT QUICKER to do a copy paste to restore large files such as maps rather than re-downloading it from the server.

Of course if we could manually assign where to save it that would be ideal!

**Reply by biodranik** [reply]
Maps should be there (`/data/data/app.organicmaps/files`) if you didn't select an SD card in OM settings.
It is a Google restriction for Android 11 and above.

**Reply by biodranik** [reply]
Btw, re-downloading should be fast enough. Usually you don't need to store the whole World on your device.
